Sources/AppsFlyerCommandTracker.swift

```diff
@@ -137,7 +137,7 @@ public class AppsFlyerCommandTracker: NSObject, AppsFlyerTrackable, TealiumRegis
 extension AppsFlyerCommandTracker: AppsFlyerTrackerDelegate {
 
     public func onConversionDataSuccess(_ conversionInfo: [AnyHashable: Any]) {
-        guard let tealium = tealium else { return }
+        guard let tealium = self.tealium else { return }
```

Comment:
I didn't get any compiler warnings/errors here. What version and build of Xcode are you on? This also might be a timing issue, since Tealium must be fully initialized at this point.